src/makeRobotAccountant.js

```diff
@@ -7,6 +7,21 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
+
+  const getSum = num1 => {
+    count++;
+
+    return num2 => {
```

Comment:
always add parentheses, even if you only have one param, it'll help with readability
```suggestion
  const getSum = (num1) => {
    count++;

    return (num2) => {
```